npe2/_types.py

```diff
@@ -31,9 +40,28 @@ def __array__(self) -> "np.ndarray":
 DataType = Union[ArrayLike, Sequence[ArrayLike]]
 FullLayerData = Tuple[DataType, Metadata, LayerName]
 LayerData = Union[Tuple[DataType], Tuple[DataType, Metadata], FullLayerData]
```

Comment:
sure, happy to do it.  I don't necessarily intend for this to be the document.  (wherever we put those efforts should be done in such a way that it can also go on the website, easily)